docs/appendix.rst

```diff
@@ -200,6 +200,24 @@ This documentation uses a few special terms to refer to Python types:
       define the use of a query language, it is specified with the string
       ``DMTF:QCL``. CQL is described in DMTF standard :term:`DSP0202`.
 
+   Schema
+      A schema (named CIM schema in the DMTF specification :term:`DSP0004`) in
+      the DMTF CIM model is a set of classes with a single defining authority
+      or owning organization. In the CIM model, the schema elements for a class
+      is defined as the components of the class name separated by the character
+      "_" and the CIM schema is the first such element.  For example ``CIM`` is
+      the schema for the class ``CIM_ManagedElement``.
```

Comment:
The term "CIM model" is not defined in DSP0004. You use it for the metamodel, but given the DSP0004 definition of "CIM schema" which uses the term "Common Information Model" it can easily be misunderstood to mean the DMTF CIM schema. I suggest not to use the term "CIM model".

Reply:
DONE